src/styles/catalog.scss

```diff
@@ -0,0 +1,12 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  padding: 50px 40px;
+  gap: 46px 48px;
+
+  @each $width, $columns in $grid-params {
+    @media (min-width: $width) {
+      grid-template-columns: repeat($columns, 200px);
```

Comment:
`200px` is a card width - possibly it is a good candidate to move to the variable